src/appleseed/renderer/kernel/intersection/embreescene.h

```diff
@@ -43,7 +43,9 @@
 #include "foundation/utility/uid.h"
 
 // Embree headers.
+#ifdef APPLESEED_WITH_EMBREE
```

Comment:
We already do the test before including this header. I don't think we need to repeat it here.